src/pygenesis/http_helper.py

```diff
@@ -83,14 +83,68 @@ def get_data_from_endpoint(endpoint: str, method: str, params: dict) -> str:
     )
 
     response = requests.get(url, params=params_, timeout=(5, 15))
-    response.encoding = "UTF-8"
 
+    # if the response requires starting a job, new params are generated and the job is started
+    try:
+        # test for job-relevant status code
+        response_status_code = response.json().get("Status").get("Code")
+
+        if response_status_code == 98:
+            logger.info(
+                "Die Daten sind zu groß um direkt geladen zu werden, es wird ein Job angestoßen."
+            )
+            # updating params to start a job
+            new_params = params.copy()
+            new_params.update({"job": "true"})
+
+            # starting a job
+            jobs_response = load_data(
+                endpoint=endpoint,
+                method=method,
+                params=new_params,
+                as_json=True,
+            )
+
+            return str(_jobs_job_id(jobs_response))
```

Comment:
Was ist die Idee hinter diesem return value? `load_data` ruft ja `get_data_from_endpoint` auf und erwartet eigentlich die Daten als raw text zurück und cacht diese. Jetzt bekommen wir hier dann eine Job-ID zurück, heißt das, wir müssen auch über das Caching in load_data nachdenken bzw. diesen Fall handeln?

Reply:
Der Name ist auch sehr irreführend. `jobs_job_id` liest zwar die Job ID aus macht damit aber gar nichts, oder? Zurückgegeben wird ja dennoch die ursprüngliche Response? Das ist irgendwie nicht sauber, warum brauchen wir das? Nur um den User über die Job-ID zu informieren?